### PR TITLE
LPS-134999 Add ImageEditorModal tests

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/plugins/browser/components/contents/components/PageContent.test.js [hunk 1]
```diff
@@ -133,6 +155,19 @@ describe('PageContent', () => {
 		).toBeInTheDocument();
 	});
 
+	it('open image editor modal when the Edit Image action is clicked', async () => {
+		const {baseElement, queryByText} = renderPageContent(contents[2]);
+
+		fireEvent.click(queryByText('open-actions-menu'));
+		fireEvent.click(queryByText('edit-image'));
+
+		await wait(() => {
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/plugins/browser/components/contents/components/PageContent.test.js:164]: Just as a note, you can use jest.runAllTimers() here and there is no need to wait for the modal to open :). It doesn't matter in this case but I can speed up the test a bit if we are opening a lot of modals

**Comment by victorg1991** [comment]
LGTM 🎉

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**